docs/network/interfaces_and_networks.md

```diff
@@ -91,61 +98,58 @@ definition.
 apiVersion: "k8s.cni.cncf.io/v1"
 kind: NetworkAttachmentDefinition
 metadata:
-  name: bridge-test
+  name: linux-bridge-net-ipam
 spec:
   config: '{
       "cniVersion": "0.3.1",
-      "name": "bridge-test",
-      "type": "bridge",
-      "bridge": "br1",
-      "disableContainerInterface": true
+      "name": "mynet",
+      "plugins": [
+        {
+          "type": "bridge",
+          "bridge": "br10",
+          "disableContainerInterface": true,
+          "ipam": {
+            "type": "whereabouts",
+            "range": "10.1.1.0/24"
+          },
+          "macspoofchk": true
+        }
+      ]
     }'
 ```
 
 With following definition, the VMI will be connected to the default pod
-network and to the secondary Open vSwitch network.
+network and to the secondary bridge network, referencing the `NetworkAttachmentDefinition` 
+shown above(in the same namespace)
 
 ```yaml
-kind: VM
+# partial example - kept short for brevity 
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
 spec:
-  domain:
-    devices:
-      interfaces:
-        - name: default
-          masquerade: {}
-          bootOrder: 1   # attempt to boot from an external tftp server
-          dhcpOptions:
-            bootFileName: default_image.bin
-            tftpServerName: tftp.example.com
-        - name: ovs-net
-          bridge: {}
-          bootOrder: 2   # if first attempt failed, try to PXE-boot from this L2 networks
-  networks:
-  - name: default
-    pod: {} # Stock pod network
-  - name: ovs-net
-    multus: # Secondary multus network
-      networkName: ovs-vlan-100
+  template:
+    spec:
+       domain:
+         devices:
+           interfaces:
+           - masquerade: {}
+             name: default
+           - bridge: {}
+             name: linux-bridge-with-ipam
+       networks:
+       - name: default
+         pod: {} # Stock pod network
+       - name: linux-bridge-with-ipam
```

Comment:
Please consider naming the network with a name other than the NAD's name - in order for it to be clear that the logical network name does not need to equal the NAD's name.